src/ui/utils.ts

```diff
@@ -0,0 +1,119 @@
+import { setEmojiShorthandCompletionField, setInlineField } from "index/import/inline-field";
```

Comment:
This is specifically task related utilities so I would put it in a file named `src/utils/task.ts`.